CHANGELOG.md

```diff
@@ -1,5 +1,42 @@
 # Change Log
 
+## [0.21.0](https://github.com/eclipse/lemminx/milestone/34?closed=1) (June 29, 2022)
+
+### Enhancements
+
+ * Support folding for <!DOCTYPE. See [#1213](https://github.com/eclipse/lemminx/issues/1213).
+ * Improve XML formatter (option to have Tags in the same line). See [#594](https://github.com/eclipse/lemminx/issues/594).
+
+### Bug Fixes
+
+ * Incorrect error range for `cvc-identity-constraint.4.1`. See [#1210](https://github.com/eclipse/lemminx/issues/1210).
+ * Resolve uri as system with XML catalog. See [#1199](https://github.com/eclipse/lemminx/issues/1199).
+ * Preserve invalid content while formatting. See [#1041](https://github.com/eclipse/lemminx/issues/1041).
+ * Bad element formatting is replaced with <null>. See [#1034](https://github.com/eclipse/lemminx/issues/1034).
+ * Lone quote inside tag leads to catastrophic formatting. See [#679](https://github.com/eclipse/lemminx/issues/679).
+ * XML format for bad processing instruction removes following element. See [#675](https://github.com/eclipse/lemminx/issues/675).
+ * Fault tolerance needs improvement. See [#305](https://github.com/eclipse/lemminx/issues/305).
+
+### Other
+
+ * Update lsp4j to 0.14.0. See [#1231](https://github.com/eclipse/lemminx/pull/1231).
+ * Add ConfigurationItem and ConfigurationParams to reflect-config.json. See [#1230](https://github.com/eclipse/lemminx/pull/1230).
+ * Add test for TreeLineTracker.getLineInformation. See [#1228](https://github.com/eclipse/lemminx/pull/1228).
+ * CodeAction for cvc-complex-type.2.4.b. See [#1218](https://github.com/eclipse/lemminx/issues/1218).
+ * DOCTYPE entities interfering with hover annotation display of tags / attributes.. See [#1212](https://github.com/eclipse/lemminx/pull/1212).
+ * Improve DOM parser memory. See [#1211](https://github.com/eclipse/lemminx/pull/1211).
+ * added folding setting to keep displaying the closing tag after folding. See [#1209](https://github.com/eclipse/lemminx/pull/1209).
+ * Improve XML scanner memory. See [#1206](https://github.com/eclipse/lemminx/pull/1206).
+ * Move to LSP4j 0.13.0. See [#1204](https://github.com/eclipse/lemminx/issues/1204).
```

Comment:
I think it makes sense to only list the fact that we updated to LSP4J 0.14.0

Reply:
fixed